### ISSUE TITLE
How to test LOCUS?

### ISSUE THREAD

**Issue by xchwang1998** [NeBula-Autonomy/LOCUS#58]
I use the command 
> roscore
then 
> tmuxp load {path_to_locus}/LOCUS/tmuxp_config/run_locus.yaml
( the path in run_locus.yaml have been revised to myself)

![Screenshot from 2022-11-01 22-50-08](https://user-images.githubusercontent.com/43316399/199262598-3e22cf8c-7a82-4e34-8dfe-a35464229982.png)

The result are as following:
![Screenshot from 2022-11-01 22-46-05](https://user-images.githubusercontent.com/43316399/199262183-1a0ec43b-6a12-4075-a189-1ce97b162701.png)

This is not same with demo.

**Comment by femust** [comment]
Have a look that you are on the first tab ```1: record_at_end``` . So click ```locus_datt0:locus``` to go to the zero tab.

![image](https://user-images.githubusercontent.com/26916586/199275191-909386bd-82db-414a-bd4a-991e0d4791b5.png)

I think in older versions of tmuxp the scripts are run in a weird way, while on the newer version it starts from the "zero tab"

Let me know if that worked for you

**Comment by xchwang1998** [comment]
> Have a look that you are on the first tab `1: record_at_end` . So click `locus_datt0:locus` to go to the zero tab.
> 
> ![image](https://user-images.githubusercontent.com/26916586/199275191-909386bd-82db-414a-bd4a-991e0d4791b5.png)
> 
> I think in older versions of tmuxp the scripts are run in a weird way, while on the newer version it starts from the "zero tab"
> 
> Let me know if that worked for you

I click `locus_datt0:locus`, and no response.
![Screenshot from 2022-11-02 09-16-12](https://user-images.githubusercontent.com/43316399/199375544-bce85777-c371-4d7d-bbfa-b226046907ac.png)

The version of tools on my computer as following:
> tmux: 2.6  
> tmuxp: 1.3.5

![Screenshot from 2022-11-02 09-50-30](https://user-images.githubusercontent.com/43316399/199376075-e315f484-33ff-4579-938d-826cbaf8efb7.png)


It's version is really old. What version do you recommend?

**Comment by MoonSheepJx** [comment]
I have the same problem now.

**Comment by xchwang1998** [comment]
I install them by:
> sudo apt install tmux
> sudo apt install tmuxp

Should I install them through source code?

**Comment by femust** [comment]
https://help.sourcelair.com/terminal/working-with-multiple-tabs/ try this

I think you guys use ubuntu 18?

**Comment by xchwang1998** [comment]
Yes my system is ubuntu 18.04. I switch to ‘locus_datt0:locus’ by hit Ctrl + B and then 0.

Here have some errors, but I had source the workspace before tumxp load XXX.yaml.
![Screenshot from 2022-11-02 11-39-14](https://user-images.githubusercontent.com/43316399/199389921-2dbdfb18-4e59-49c1-b345-0a645ed4edb1.png)

**Comment by femust** [comment]
put in ~/.bashrc the path to the workspace

```
source ~/workspace/locus_ws/devel/setup.bash
```
in your case, or something like this.

**Comment by xchwang1998** [comment]
> put in ~/.bashrc the path to the workspace
> 
> ```
> source ~/workspace/locus_ws/devel/setup.bash
> ```
> 
> in your case, or something like this.

That's OK! Thanks! By the way, why do we add this command in ~/.bashrc?
> ```
> source ~/workspace/locus_ws/devel/setup.bash
> ```
I try in locus_ws use this command before 'tumxp load' and it didn't work.
> ```
> cd ~/workspace/locus_ws
> source devel/setup.bash
> tmuxp load {path_to_locus}/LOCUS/tmuxp_config/run_locus.yaml
> ```
